temba/utils/analytics/base.py

```diff
@@ -0,0 +1,17 @@
+import zope.interface
```

Comment:
Doesn't Python3 have native interfaces that would work here?

Reply:
Yeah I see that abstract base classes got an upgrade in 3.4 , they look much better now, let me try to use those and remove the dependancy.